.github/workflows/test-crop.yml

```
@@ -0,0 +1,28 @@
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+          cache: maven
+
+      - name: Run Tests with Maven
+        run: mvn test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)